src/Servers/Kestrel/test/InMemory.FunctionalTests/Http3/Http3TestBase.cs

```diff
@@ -143,385 +158,193 @@ protected void CreateConnection()
             _connection = new Http3Connection(httpConnectionContext);
             _mockTimeoutHandler.Setup(h => h.OnTimeout(It.IsAny<TimeoutReason>()))
                            .Callback<TimeoutReason>(r => _connection.OnTimeout(r));
+            _ = ReadIncomingConnections();
```

Comment:
Shouldn't the Http3Connection itself be accepting the incoming connections? It seems weird to me that unlike our Http1ConnectionTests and Http2ConnectionTests, none of the HTTP/3 tests seem to call ProcessRequestsAsync.

Reply:
This is accepting unidirectional streams from the server to the client, so it's incoming in terms of the client. We are still calling InnerProcessRequestsAsync (which is a call for ProcessRequestsAsync minus some limit checks) for Http/3 here: https://github.com/dotnet/aspnetcore/blob/03a96c6d553221ecd71221f69e908ad4c10cd964/src/Servers/Kestrel/test/InMemory.FunctionalTests/Http3/Http3TestBase.cs#L125